### ISSUE TITLE
Broomstick Spell [volatus] not working

### ISSUE THREAD

**Issue by deb4life** [Azami7/Ollivanders2#516]
Hey! How are you @Azami7 
I've been noticing whenever I'm trying to enchant a broomstick in versions after [2.6.3] its not working. I'm currently using the latest version; 2.7.2 and it wasn't working so I tried different versions and the spell is working for versions 2.6.3 and below.
Can you please look on this issue. Thank you.

https://user-images.githubusercontent.com/80181320/184439853-2bd91d57-9b60-404b-85db-60c36fd3aeaa.mp4

https://user-images.githubusercontent.com/80181320/184439917-fd9fb1e6-f979-4143-856e-15fa6c7fdad5.mp4

**Comment by Azami7** [comment]
Is that an actual broom object or a normal shovel? In newer versions of the plugin, you can only enchant a broom item given by the plugin.

**Comment by deb4life** [comment]
@Azami7 > Is that an actual broom object or a normal shovel? In newer versions of the plugin, you can only enchant a broom item given by the plugin.


I changed the object for the broomstick from a stick to a shovel. I changed it back to a stick but its still not working.

**Comment by Azami7** [comment]
That is not what I am asking. Are you trying to use a standard minecraft item and expecting it to be a broom? If so, that will not work. You need to get the broom item from the plugin using the /olli items command. A normal MC item will not be recognized as a broom. This is to prevent players "creating" items with basic crafting.

**Comment by deb4life** [comment]
> @Azami7  That is not what I am asking. Are you trying to use a standard minecraft item and expecting it to be a broom? If so, that will not work. You need to get the broom item from the plugin using the /olli items command. A normal MC item will not be recognized as a broom. This is to prevent players "creating" items with basic crafting.


No, I'm not using a standard minecraft shovel and expecting the plugin to recognize it as a broom. 
In the config i changed the line...
"broomstick: STICK"
to 
"broomstick: SHOVEL"

so whenever i use the command /olli item give 1 broomstick it gives me a Broomstick in the form of a **minecraft shovel** instead of a stick from the original config because I changed it to that because i wanted it to be different from the wand.

And using either the "stick" or "shovel" its not working even if i want to change the broomstick item to something else.

**Comment by Azami7** [comment]
If you turn on debug, what shows up in your logs when you hold a broom and then stop holding it? I am curious if it shows the message "<player name> is holding a broom". If so, then it means it is properly detecting the item is a broomstick but flying is not being set or if it doesn't even think the item is a broomstick in the first place.

**Comment by deb4life** [comment]
> If you turn on debug, what shows up in your logs when you hold a broom and then stop holding it? I am curious if it shows the message " is holding a broom". If so, then it means it is properly detecting the item is a broomstick but flying is not being set or if it doesn't even think the item is a broomstick in the first place.

I figure out what happened, In the versions 2.6.3 and previous, when i pick up the broom after enchanting it automatically allows me to float and i now release that i have to double click like im in creative to fly becuase i viewed the debug and said that i got the flying effect. Thanks.

**Comment by deb4life** [comment]
So @Azami7 I had an issue with this mode of transportation and now i have an issue with the portkey 🥲. If i cast it at a floating item on the ground as instructed, it only detects the blocks placed underneath the item.

CONSOLE:
[16:23:28] [Async Chat Thread - #15/INFO]: <[God] deb4life> portus 225 27 558
[16:23:28] [Server thread/INFO]: [Ollivanders2] Checking GRASS_BLOCK

**Comment by Azami7** [comment]
Glad the broom is working. I'll take a look at the portkeys.
